tests/e2e/scripts/daiosmov16.json

```diff
@@ -0,0 +1,7 @@
+{
+    "weights": "5udai,5uosmo",
+    "initial-deposit": "499404udai,500000uosmo",
```

Comment:
side note, we will likely want to use these pools in the future in e2e to test things like ensuring superfluid stakes migrate properly, so having more than .5dai and .5osmo might be helpful

Reply:
increased to 5k for now. For higher amounts, we might have to change e2e init image, so let's come back to that if / when needed